src/scripts/plugincatalogpage.js

```diff
@@ -1,78 +1,206 @@
-define(["loading", "libraryMenu", "globalize", "cardStyle", "emby-linkbutton", "emby-checkbox", "emby-select"], function(loading, libraryMenu, globalize) {
+define(["loading", "libraryMenu", "globalize", "cardStyle", "emby-linkbutton", "emby-checkbox", "emby-select"], function (loading, libraryMenu, globalize) {
     "use strict";
 
     function reloadList(page) {
-        loading.show(), query.IsAppStoreSafe = !0;
-        var promise1 = ApiClient.getAvailablePlugins(query),
-            promise2 = ApiClient.getInstalledPlugins();
-        Promise.all([promise1, promise2]).then(function(responses) {
+        loading.show();
+        query.IsAppStoreSafe = true;
+        var promise1 = ApiClient.getAvailablePlugins(query);
+        var promise2 = ApiClient.getInstalledPlugins();
+        Promise.all([promise1, promise2]).then(function (responses) {
             populateList({
                 catalogElement: page.querySelector("#pluginTiles"),
                 noItemsElement: page.querySelector("#noPlugins"),
                 availablePlugins: responses[0],
                 installedPlugins: responses[1]
-            })
-        })
+            });
+        });
     }
 
     function populateList(options) {
-        populateListInternal(options)
+        populateListInternal(options);
     }
 
     function getHeaderText(category) {
         category.replace(" ", "").replace(" ", "");
```

Comment:
wtf does this double `replace` mean, isn't one enough?

Reply:
It's a no-op essentially. String.replace() returns a new string :man_shrugging: